crates/trigger-redis/src/lib.rs

```diff
@@ -99,9 +99,14 @@ impl TriggerExecutor for RedisTrigger {
             .collect();
 
         // wait for the first handle to be returned and drop the rest
-        let (_, _, rest) = futures::future::select_all(tasks).await;
+        let (result, _, rest) = futures::future::select_all(tasks).await;
+
         drop(rest);
 
+        if let Err(err) = result? {
+            return Err(err);
+        }
```

Comment:
I am guessing this stanza and the next line could be replaced by something like

```
result?
```

Or maybe only this stanza and it's something like `result??;`

Just seems little point decomposing the Err case just to put it back together again.

Reply:
Yep changed it to that.